mitiq/calibration/settings.py

```diff
@@ -371,4 +396,56 @@ def make_strategies(self) -> List[Strategy]:
     ],
 )
 
+PECSettings = Settings(
+    benchmarks=[
+        {
+            "circuit_type": "ghz",
+            "num_qubits": 2,
+        },
+        {
+            "circuit_type": "w",
+            "num_qubits": 2,
+        },
+        {
+            "circuit_type": "rb",
+            "num_qubits": 2,
+            "circuit_depth": 7,
+        },
+        {
+            "circuit_type": "mirror",
+            "num_qubits": 2,
+            "circuit_depth": 7,
+            "circuit_seed": 1,
+        },
+    ],
+    strategies=[
+        {
+            "technique": "pec",
+            "representation_function": (
+                represent_operation_with_local_depolarizing_noise
+            ),
+            "operations": [
+                Circuit(CNOT(*LineQubit.range(2))),
+                Circuit(CZ(*LineQubit.range(2))),
+            ],
+            "is_qubit_dependent": False,
+            "noise_level": 0.001,
+            "num_samples": 200,
+        },
```

Comment:
Do you think we should validate each instance that comes in through the strategy list? Probably a bigger change than we'd want to make here, but curious on your thoughts.

Reply:
Do we do that for ZNE?

Reply:
Nope!

Reply:
Then I suggest to address in a future PR, issue #1862 created.